flutter-for-react-native/index.md

```diff
@@ -0,0 +1,2538 @@
+`cupertino_icons` in your project's `pubspec.yaml` file.
+<!-- skip -->
+```yaml
+name: my_awesome_application
+dependencies:
+  cupertino_icons: ^0.1.0
+```
+
+For more information and to view the available icons, see [`Icons`](https://docs.flutter.io/flutter/material/Icons-class.html).
+
+To customize the colors and styles of Material Components you can pass in a
+ `ThemeData` object to the `MaterialApp` widget then use the  [`Colors`](https://docs.flutter.io/flutter/material/Colors-class.html)
+ class which uses the Material Design [color palette](https://material.io/guidelines/style/color.html).
+
+
+ The example below shows the primary swatch is set to `blue` and all text
+ selection color is set to `red`.
+<!-- skip -->
+{% prettify dart %}
+class SampleApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'Sample App',
+      theme: new ThemeData(
+        [[highlight]]primarySwatch: Colors.blue,[[/highlight]]
+        [[highlight]]textSelectionColor: Colors.red[[/highlight]]
+      ),
+      home: new SampleAppPage(),
+    );
+  }
+}
+{% endprettify %}
+
+
+
+### How do I add style themes?
+
+In React Native, common themes are defined for components in stylesheets and
+then used in components.
+
+In Flutter you can add uniform styling for almost everything by defining the
+ styling in the [`ThemeData`](https://docs.flutter.io/flutter/material/ThemeData-class.html) class and passing it to the theme property
+ in the [`MaterialApp`](https://docs.flutter.io/flutter/material/MaterialApp-class.html) widget. The `MaterialApp` widget wraps the other
+ widgets and acts as the app root widget.
+
+<!-- skip -->
+```Dart
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      theme: new ThemeData(
+        primaryColor: Colors.cyan,
+        brightness: Brightness.dark,
+      ),
+      home: new StylingPage(),
+    );
+  }
+```
+
+A `Theme` can be applied even without using the `MaterialApp` widget. The
+ [`Theme`](https://docs.flutter.io/flutter/material/Theme-class.html) widget takes a `ThemeData` in its `data` parameter and applies the
+ `ThemeData` to all of its children widgets.
+
+<!-- skip -->
+```Dart
+ @override
+  Widget build(BuildContext context) {
+    return new Theme(
+      data: new ThemeData(
+        primaryColor: Colors.cyan,
+        brightness: brightness,
+      ),
+      child: new Scaffold(
+         backgroundColor: Theme.of(context).primaryColor,
+              ...
+              ...
+      ),
+    );
+  }
+```
+
+## State Management
+State is information that can be read synchronously when a widget is built or information that might change during the lifetime of a widget. To manage app state in Flutter, use the
+ [StatefulWidget](https://docs.flutter.io/flutter/widgets/StatefulWidget-class.html) and the [StatelessWidget](https://docs.flutter.io/flutter/widgets/StatelessWidget-class.html).
```

Comment:
To manage app state in Flutter, use the StatefulWidget and StatelessWidget.

Nope. Delete StatelessWidget from that line:
To manage app state in Flutter, use a StatefulWidget paired with a State object.

Reply:
Done